application/frontend/config/main.php

```diff
@@ -1,32 +1,43 @@
 <?php
 
+use Sil\Idp\IdSync\common\components\ApiConsumer;
```

Comment:
I noticed much more namespacing, wondering if I need to do the same on the broker?